src/MissionManager/MissionCommandUIInfo.cc

```diff
@@ -364,7 +364,7 @@ bool MissionCommandUIInfo::loadJsonInfo(const QJsonObject& jsonObject, bool requ
 
             // Validate key types
             QList<QJsonValue::Type> types;
-            types << QJsonValue::Null << QJsonValue::Double << QJsonValue::String << QJsonValue::String
+            types << QJsonValue::Double << QJsonValue::Double << QJsonValue::String << QJsonValue::String
```

Comment:
The default value can be either a Double or a NaN. Which is signified by the `QJsonValue::Null` type. So this is correct as is.